src/sentry/eventtypes/__init__.py

```diff
@@ -1,3 +1,13 @@
+__all__ = [
+    "CspEvent",
+    "DefaultEvent",
+    "ErrorEvent",
+    "ExpectCTEvent",
+    "ExpectStapleEvent",
+    "HpkpEvent",
+    "TransactionEvent",
+]
```

Comment:
I would remove this personally -- it's better to import explicitly since `__init__.py` makes it much easier to introduce import cycles and to worsen startup performance